R/rocRAND/build_tarballs.jl

```diff
@@ -18,13 +18,17 @@ cd ${WORKSPACE}/srcdir/rocRAND*/
 amdgpu_targets="gfx900,gfx906,gfx908,gfx1010,gfx1011,gfx1012"
 
 mkdir build && cd build
+ln -s ${prefix}/bin/clang ${prefix}/tools/clang
 export ROCM_PATH=${prefix}
 export HIP_CLANG_PATH=${prefix}/tools
 export HIP_CLANG_HCC_COMPAT_MODE=1
-export HIP_RUNTIME=ROCclr
+export HIP_RUNTIME=rocclr
 export HIP_COMPILER=clang
-export HIP_PLATFORM=rocclr
-export HIPCC_VERBOSE=1
+export HIP_PLATFORM=amd
+export HIP_ROCCLR_HOME=${prefix}/lib
+export HIPCC_VERBOSE=7
+export HIP_LIB_PATH=${prefix}/hip/lib
+export LD_LIBRARY_PATH=${prefix}/hip/lib:$LD_LIBRARY_PATH
```

Comment:
Interesting 😬 why is that needed? I presume it works only because we're targeting platforms for which we happen to be able to do native build?

Reply:
Sorry, artifact from trying to get this to build. Removed in latest push.